package-lock.json

```diff
@@ -417,9 +452,41 @@
       "dev": true
     },
     "@types/node": {
-      "version": "14.18.21",
-      "resolved": "https://registry.npmjs.org/@types/node/-/node-14.18.21.tgz",
-      "integrity": "sha512-x5W9s+8P4XteaxT/jKF0PSb7XEvo5VmqEWgsMlyeY4ZlLK8I6aH6g5TPPyDlLAep+GYf4kefb7HFyc7PAO3m+Q==",
+      "version": "14.17.32",
+      "resolved": "https://registry.npmjs.org/@types/node/-/node-14.17.32.tgz",
+      "integrity": "sha512-JcII3D5/OapPGx+eJ+Ik1SQGyt6WvuqdRfh9jUwL6/iHGjmyOriBDciBUu7lEIBTL2ijxwrR70WUnw5AEDmFvQ=="
+    },
+    "@types/puppeteer": {
+      "version": "5.4.6",
+      "resolved": "https://registry.npmjs.org/@types/puppeteer/-/puppeteer-5.4.6.tgz",
+      "integrity": "sha512-98Kghehs7+/GD9b56qryhqdqVCXUTbetTv3PlvDnmFRTHQH0j9DIp1f7rkAW3BAj4U3yoeSEQnKgdW8bDq0Y0Q==",
+      "dev": true,
+      "requires": {
+        "@types/node": "*"
+      }
+    },
+    "@types/puppeteer-core": {
+      "version": "5.4.0",
+      "resolved": "https://domoreexp.pkgs.visualstudio.com/_packaging/teams-verticals/npm/registry/@types/puppeteer-core/-/puppeteer-core-5.4.0.tgz",
```

Comment:
Looks like the reference here to a private ADO feed `@types/puppeteer-core` is what's causing the PR Build break

Reply:
wow... thanks for finding the cause of the issue.